mix.exs

```diff
@@ -97,7 +97,7 @@ defmodule Membrane.RTC.Engine.MixProject do
   defp docs do
     [
       main: "readme",
-      extras: ["README.md", "LICENSE"],
+      extras: ["README.md", "media_events.md", "LICENSE"],
```

Comment:
let's put it under `doc/` similar to pheonix https://github.com/phoenixframework/phoenix/blob/master/mix.exs